docs/intro/quick-start/reading-from-network.md

```diff
@@ -129,6 +129,7 @@ Let's break down the fields in the output:
 - `data` - This field contains what we generally refer to as the account "data".
   For a wallet, it's empty (0 bytes), but other accounts use this field to store
   any arbitrary data as a serialized buffer of bytes.
+      *Note*: When data is "buffered" in this way, it maintains its integrity and can be later deserialized back into its original form for use in applications. This process is widely used in blockchain for efficient data handling.
```

Comment:
Due to the way the solana-com website will handle/render this, can you update the formatting:

```suggestion
> When data is "buffered" in this way, it maintains its integrity and can be
> later deserialized back into its original form for use in applications. This
> process is widely used in blockchain for efficient data handling.
```


--- 
This is what it will render as with my suggested changes: 
---
![image](https://github.com/user-attachments/assets/3f4db6f6-a487-43f3-b174-55085622aaaf)